### PR TITLE
Nft collection test

### PR DESCRIPTION
**Pull request by shalinikum**

### CODE REVIEW

File: 0chain-0box.postman_collection .json [hunk 1]
```diff
@@ -0,0 +1,5955 @@
+{
```
Comment (Kishan-Dhakan) [inline 0chain-0box.postman_collection .json:1]: Remove this file

File: 0chain-0box.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,6027 @@
+{
```
Comment (Kishan-Dhakan) [inline 0chain-0box.postman_collection.json:1]: Remove this file

File: internal/api/util/client/zbox_client.go [hunk 3]
```diff
@@ -27,7 +28,7 @@ func NewZboxClient(zboxEntrypoint, defaultPhoneNumber string) *ZboxClient {
...
+		DefaultAuthTicket:     "eyJjbGllbnRfaWQiOiIiLCJvd25lcl9pZCI6ImEzMzQ1NGRhMTEwZGY0OTU2ZDc1YzgyMDA2N2M1ZThmZTJlZjIyZjZkNWQxODVhNWRjYTRmODYwMDczNTM1ZDEiLCJhbGxvY2F0aW9uX2lkIjoiZTBjMmNkMmQ1ZmFhYWQxM2ZjNTM3MzNkZDc1OTc0OWYyYjJmMDFhZjQ2MzMyMDA5YzY3ODIyMWEyYzQ4ODE1MyIsImZpbGVfcGF0aF9oYXNoIjoiZTcyNGEyMjAxZTIyNjUzZDMyMTY3ZmNhMWJmMTJiMmU0NGJhYzYzMzdkM2ViZGI3NDI3ZmJhNGVlY2FhNGM5ZCIsImFjdHVhbF9maWxlX2hhc2giOiIxZjExMjA4M2YyNDA1YzM5NWRlNTFiN2YxM2Y5Zjc5NWFhMTQxYzQwZjFkNDdkNzhjODNhNDk5MzBmMmI5YTM0IiwiZmlsZV9uYW1lIjoiSU1HXzQ4NzQuUE5HIiwicmVmZXJlbmNlX3R5cGUiOiJmIiwiZXhwaXJhdGlvbiI6MCwidGltZXN0YW1wIjoxNjY3MjE4MjcwLCJlbmNyeXB0ZWQiOmZhbHNlLCJzaWduYXR1cmUiOiIzMzllNTUyOTliNDhlMjI5ZGRlOTAyZjhjOTY1ZDE1YTk0MGIyNzc3YzVkOTMyN2E0Yzc5MTMxYjhhNzcxZTA3In0=",
```
Comment (Kishan-Dhakan) [inline internal/api/util/client/zbox_client.go:31]: Add //nolint:revive here or try using raw string

File: internal/api/util/client/zbox_client.go [hunk 4]
```diff
@@ -708,3 +710,228 @@ func (c *ZboxClient) ContactWallet(t *test.SystemTest, reqBody, idToken, csrfTok
 
 	return resp, err
 }
+
+func (c *ZboxClient) CreateNftCollectionId(t *test.SystemTest, idToken, csrfToken, phoneNumber, createdBy, collectionName, collectionId, totalNfts, collectionType, allocationId, baseUrl, symbol string, pricePerPack, maxMints, CurrMints, batchSize int) (*model.ZboxNftCollection, *resty.Response, error) {
```
Comment (Kishan-Dhakan) [inline internal/api/util/client/zbox_client.go:714]: Rename CurrMints to currMints

File: tests/api_tests/0box_test.go [hunk 5]
```diff
@@ -13,13 +12,336 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func Test0BoxNFT(testSetup *testing.T) {
```
Comment (Kishan-Dhakan) [inline tests/api_tests/0box_test.go:15]: Please cleanup this file once you're done with local testing

File: tests/api_tests/0box_test.go [hunk 6]
```diff
@@ -13,13 +12,336 @@ import (
...
+}
+
 func Test0Box_share_info(testSetup *testing.T) {
 	// todo: These tests are sequential and start with teardown as they all share a common phone number
 	t := test.NewSystemTest(testSetup)
 
 	firebaseToken := authenticateWithFirebase(t, zboxClient.DefaultPhoneNumber)
 
 	t.RunSequentially("Post ShareInfo with correct AuthTicket should work properly", func(t *test.SystemTest) {
+
```
Comment (Kishan-Dhakan) [inline tests/api_tests/0box_test.go:344]: remove unnecessary whitespaces

File: tests/api_tests/0box_test.go [hunk 7]
```diff
@@ -599,311 +929,6 @@ func Test0Box(testSetup *testing.T) {
 	// [{ClientID  required } {PublicKey  required } {Timestamp  required } {TokenInput  required } {AppType  required } {PhoneNumber  required }]
 }
 
-func Test0BoxWallet(testSetup *testing.T) {
```
Comment (Kishan-Dhakan) [inline tests/api_tests/0box_test.go:602]: This should not be removed, might be branching issue, please re-add this

File: tests/api_tests/config/api_tests_config.yaml [hunk 8]
```diff
@@ -1,4 +1,9 @@
+<<<<<<< HEAD
```
Comment (Kishan-Dhakan) [inline tests/api_tests/config/api_tests_config.yaml:1]: Remove the conflict lines please. Also, remove the 2 png files added in the PR if unnecessary

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

**Comment by service-0chain** [comment]
Manual system tests [cancelled] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | ABILITY TO SKIP TESTS IS DISABLED. SKIP_TESTS OPTION IGNORED.  |
| system-tests  | system_test_513  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | ABILITY TO SKIP TESTS IS DISABLED. SKIP_TESTS OPTION IGNORED.  |
| system-tests  | system_test_513  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Review by Kishan-Dhakan — APPROVED** [review]

**Pull request merged**